.claude/skills/vdct-conversion/rename-vdct-orphans.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Rename VDCT-tagged source files to .vdb so vdct2template's *.vdb glob sees them.
+# EXCLUDES files named inside an expand() block - those must keep their .template
+# name or the reference breaks.
+shopt -s nullglob
+cd "$1" || exit 1
+refs=$(grep -ho 'expand("[^"]*"' * 2>/dev/null | sed 's/expand("//;s/"//' | sort -u)
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)